doc/sphinx/language/core/primitive.rst

```diff
@@ -171,3 +171,48 @@ the array are modified have O(n) access time, the same as a list. The version th
 changes dynamically upon each get and set call: the current list of modifications
 is applied to the native array and the lists of modifications of the other versions
 are updated so that they still represent the same values.
+
+.. _primitive-string:
+
+Primitive Strings
+----------------
+
+The language of terms features immutable strings as values. The type of such a
+value is *axiomatized*; it is declared through the following sentence (excerpt
+from the :g:`PrimString` module):
+
+.. coqdoc::
+
+   Primitive string := #string_type.
```

Comment:
How about making this a ..coqtop instead, which will check the syntax?

Reply:
Let's not do that for consistency with the other sections.